### PR TITLE
Week3 | Part1

### PR DESCRIPTION
**Pull request by IanCStewart**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+
+@font-face {
+  font-family: 'Lato';
+  src: url('./assets/fonts/Lato/Lato-Light.ttf');
+  font-weight: lighter;
+  font-style: normal;
+}
+
+html {
+  font-size: 62.5%;
```
Comment (rijkvanzanten) [inline index.css:23]: This resets font-size to 10px, but you can't know for certain that the default font size is 16px

File: index.css [hunk 2]
```diff
@@ -0,0 +1,457 @@
...
+  --text-large: calc(var(--text-body) * 2.25);
+  --text-medium: calc(var(--text-body) * 1.75);
+  --text-small: calc(var(--text-body) * 1.375);
+  --text-body: 1.6rem;
+  --text-xs: 1.2rem;
+}
+
+html,
+body {
+  width: 100vw;
```
Comment (rijkvanzanten) [inline index.css:45]: Isn't this default?

File: index.css [hunk 3]
```diff
@@ -0,0 +1,457 @@
...
+  height: 100%;
+  margin: 0;
+}
+
+* {
```
Comment (rijkvanzanten) [inline index.css:50]: You can also apply these rules to `html` instead of `*` (= little performance increase)

File: index.css [hunk 4]
```diff
@@ -0,0 +1,457 @@
...
+
+nav li > ul a:hover,
+nav li > ul a:focus {
+  background: #efefef;
+}
+
+/*-- Media Items --*/
+.media-items {
+  padding: 2.6rem 1.6rem;
+  display: -webkit-flex;
```
Comment (rijkvanzanten) [inline index.css]: Why would you prefix this?
http://caniuse.com/#feat=flexbox

File: index.css [hunk 5]
```diff
@@ -0,0 +1,457 @@
...
+.media-items article:hover > a,
+.media-items article:active > a,
+.media-items a:focus {
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+  opacity: 1;
+}
+
+/*-- Login -- */
+#login {
```
Comment (rijkvanzanten) [inline index.css:277]: Try to defer using id selectors, specificity can and will f*ck you over in the long run

File: index.css [hunk 6]
```diff
@@ -0,0 +1,457 @@
...
+  font-size: var(--text-small);
+  color: var(--color-beige);
+  position: relative;
+  top: -9px;
+  left: 4px;
+  opacity: 0;
+  transition: all .3s ease-in-out;
+}
+
+form[name="login"] > label[for*="stay"] > input:checked + span::before,
```
Comment (rijkvanzanten) [inline index.css]: Couldn't this selector be written slightly less complicated?

File: index.css [hunk 7]
```diff
@@ -0,0 +1,457 @@
...
+}
+
+form[name="login"] > button[name="close-login"] {
+  position: absolute;
+  top: 1rem;
+  right: 1rem;
+  background: none;
+  border-radius: 50%;
+  margin: 0;
+  z-index: +1;
```
Comment (rijkvanzanten) [inline index.css:407]: `+1` ? 🤔

File: index.html [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+  <link rel="stylesheet" href="./index.css">
+  <script type="text/javascript" src="./index.js" defer></script>
+</head>
+<body>
+  <header class="side-nav">
+    <input type="checkbox" id="hamburger-menu" name="hamburger-menu">
```
Comment (rijkvanzanten) [inline index.html:13]: This could be an accessibility problem

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: index.css (hunk 2 above)
Comment (IanCStewart) [reply index.css:45]: safety for the win

File: index.css (hunk 1 above)
Comment (IanCStewart) [reply index.css:23]: risk i'm willing to take

File: index.css (hunk 7 above)
Comment (IanCStewart) [reply index.css:407]: it's a thing

File: index.html (hunk 8 above)
Comment (IanCStewart) [reply index.html:13]: i don't see why

**Pull request merged**